packages/charts/.gitignore

```diff
@@ -3,3 +3,4 @@ sdk/
 dist/
 **/coverage/
 .tmp
+_snapshots
```

Comment:
Just curious, but what are they snapshots?

Reply:
This is from before the merge from main which already has this change. Lukas added this in his PR. It's the directory that gets created when the Percy visual test build task is run.